data/bitmap_font_data.c

```diff
@@ -0,0 +1,26 @@
+#include "bitmap_font_data.h"
+
+extern const char D_0804f338[];
+
+const char *D_08936b50 = D_0804f338;
+
+struct TaskMethods D_08936b54 = {
+    (TaskStartFunc)bmp_font_obj_init_multi_printer,
```

Comment:
these casts can be removed by changing the struct definition slightly, i think you just do () (no void) for the parameters, lmk if you cant figure it out

Reply:
Doesn't immediately work because TaskStartFunc has a distinct `void *` return value. Perhaps if we wanted to edit the definitions of every single task-starting function we could do this.